FrontEnd/src/components/Header/Navbar/Menu.jsx

```diff
@@ -3,23 +3,14 @@ import css from './Menu.module.css';
 import { HashLink } from 'react-router-hash-link';
 
 const MENU_LINKS = [
-  {
-    id: 'm0',
-    title: 'Головна',
-    link: '/',
-  },
-  {
-    title: 'Компанії',
-    link: '/profiles/companies',
-  },
-  {
-    title: 'Стартапи',
-    link: '/profiles/startups',
-  },
   {
     title: 'Про нас',
     link: '/#about-us',
   },
+  {
+    title: 'Підприємства та сектори',
+    link: '/profiles/companies',  // SHOULD BE CHANGED FOR COMPANY LIST (Підприємства та сектори)
```

Comment:
Could we resolve this `TODO`?

Reply:
Unfortunatelly we cannot for now, because the whole page, where the link should direct, is not done yet